src/Employee.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Objects;
+
+public class Employee {
+    private String FullName;
```

Comment:
```suggestion
    private String fullName;
```
названия полей надо писать с маленькой буквы